src/graph/LookupExecutor.cpp

```diff
@@ -491,7 +511,14 @@ std::vector<std::shared_ptr<nebula::cpp2::IndexItem>> LookupExecutor::findIndexF
 void LookupExecutor::lookUp() {
     auto *sc = ectx()->getStorageClient();
     auto filter = Expression::encode(sentence_->whereClause()->filter());
-    auto future  = sc->lookUpIndex(spaceId_, index_, filter, returnCols_, isEdge_);
+    // Get the columns that really need to be requested
+    std::vector<std::string> requestCols_;
+    for (auto& column : returnCols_) {
+        if (column != _SRC && column != _DST) {
+            requestCols_.emplace_back(column);
+        }
+    }
+    auto future  = sc->lookUpIndex(spaceId_, index_, filter, requestCols_, isEdge_);
```

Comment:
How to identify `_ID` of vertex?

Reply:
This PR only supports specifying  and outputing the start and end vertice ID of the edge in the Yield clause, and It's also consistent with the implementation of Go Sentence. 
Actually, the judgment of `prop->kind()` in `prepareYield()` method has ensured that it is illegal to specify the vertex id, so there is no need to judge this situation here.

Reply:
> This PR only supports specifying and outputing the start and end vertice ID of the edge in the Yield clause, and It's also consistent with the implementation of Go Sentence.
> Actually, the judgment of `prop->kind()` in `prepareYield()` method has ensured that it is illegal to specify the vertex id, so there is no need to judge this situation here.

Got, excellent.